src/native/ceph_s3select/ceph_s3lect_napi.cpp

```diff
@@ -0,0 +1,428 @@
+/* Copyright (C) 2016 NooBaa */
+
+#include "../../../submodules/ceph_s3select/include/s3select.h"
+#include "../util/common.h"
+#include "../util/napi.h"
+#include "../../../submodules/boost/libs/crc/include/boost/crc.hpp"
+#include <arpa/inet.h>
+
+namespace noobaa {
+
+DBG_INIT(0);
+
+enum InputFormat {csv, json, parquet};
+
+class awsCli_handler {
```

Comment:
move to js. also this name seems very confusing - how is this related to awscli??